wikipedia-dark.user.css

```diff
@@ -1974,10 +1974,6 @@ regexp("https?:\/\/wiki\.(archlinux|mozilla)\.(org|jp)\/.*$") {
   td[style*="background: #F5F5FF"] {
     background-color: var(--gray-28) !important;
   }
-  html .thumbimage {
-    background-color: var(--gray-28);
-    border-color: var(--gray-5);
-  }
```

Comment:
you sure that archlinux/mozilla doesnt have this? It woulnt be here otherwise.

Reply:
This is redundant. You can check line 2868. It is already there. Also, the background-color actually makes the text on svg become unreadable. So I removed it.

Reply:
I dont mind if this breaks something, Im sure its bound to break something you aren't intending with the removal, also I dont really have time to go investigate which isnt helpful but there is #151 where its clear I dont have time for these side things =(

so @AfroThundr3007730 if he has time please, if not idk.